docs/spec/staking/state.md

```diff
@@ -152,29 +149,26 @@ type UnbondingDelegationEntry struct {
 
 ### Redelegation
 
-Shares in a `Delegation` can be rebonded to a different validator, but they must
-for some time exist as a `Redelegation`, where shares can be reduced if Byzantine
-behavior is detected. This is tracked as moving a delegation from a `ValidatorSrcAddr`
-to a `ValidatorDstAddr`.
+The bonded tokens worth of a `Delegation` may be instantly redelegated from a
+source validator to a different validator (destination validator). However when
+this occurs they must be tracked in a `Redelegation` object, whereby their
+shares can be slashed if their tokens were found to contribute to Byzantine
+behavior in the source validator.  This is tracked as moving a delegation from
```

Comment:
```suggestion
fault committed by the source validator.  This is tracked as moving a delegation from
```

Reply:
"Moving a delegation" is confusing and doesn't reflect the implementation - can we just say "unbond and rebond" or something?

Reply:
first suggested added

Reply:
I think the whole second sentence is actually unnecessary - removing.